lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -262,6 +262,10 @@ void LocalSessionManagerHandlerImpl::send_create_session(
             Status(
               grpc::FAILED_PRECONDITION, "Failed to initialize session");
         } else {
+          
+          // Let's try to add apn tagging after main session rules are installed
+          // bool success = enforcer_->init_apn_tagging(imsi, sid, cfg, response);
```

Comment:
Not the right place to do this. Check out inside the function "send_create_session". If session initialization succeeds, then initializing the tagging should be done.

Reply:
Also if this optional feature is not enabled, return value should be more nuanced then a bool.

Reply:
@ulaskozat yes, it makes sense for me.  Will send updated patch shortly.

Reply:
Delete these lines now?